pkg/config/env.go

```diff
@@ -41,6 +41,8 @@ type DaemonConfig struct {
 	Tokens                []string        `toml:"tokens"`
 	SlackWebhookURL       string          `toml:"slack_webhook_url"`
 	GithubRepoStatusToken string          `toml:"github_repo_status_token"`
+	RootURL               string          `toml:"root_url"`
```

Comment:
RootURL is used for some links on the dashboard. When run locally it should be set to:
```
root_url = "http://localhost:8042"
```

Whereas on a given deployment, such as TaaS, it is set to:
```
    root_url = "https://ci.testground.ipfs.team"
```